src/components/shared/SEOMetaTag/index.tsx

```diff
@@ -14,22 +14,28 @@ const MetaTag = (props: Props) => {
 			<title>{props.title ? props.title : '알티모빌리티 - 차량대여 서비스'}</title>
 			<meta
 				name="description"
-				content={props.description ? props.description : '알티모빌리티 - 차량대여 서비스'}
+				content={props.description ? props.description : '세상과 사람, 자동차의 교집합'}
 				data-react-helmet="true"
 			/>
-			<meta name="keywords" content="자동차" />
+			<meta name="keywords" content="자동차, Freemium, Sharing, Big Data, Economy" />
 			<meta property="og:type" content="website" />
 			<meta property="og:title" content={props.title ? props.title : '알티모빌리티 - 차량대여 서비스'} />
 			<meta property="og:site_name" content="알티모빌리티 - 차량대여 서비스" />
 			<meta property="og:description" content={props.description ? props.description : '알티모빌리티 - 차량대여 서비스'} />
-			<meta property="og:image" content={props.imgsrc ? props.imgsrc : ''} />
+			<meta
+				property="og:image"
+				content={props.imgsrc ? props.imgsrc : 'https://static.wanted.co.kr/images/wdes/0_4.1efa9680.png'}
+			/>
 			<meta property="og:url" content={props.url ? props.url : 'https://www.naver.com/'} />
```

Comment:
`<meta property="og:url" content="웹페이지 URL" />` 이라고 하네요 ~ 그럼 저희 같은 경우는

```suggestion
			<meta property="og:url" content={props.url ? props.url : 'https://www.naver.com/'} /> // 배포될 사이트 도메인으로 default URL 을 설정
```